salt_stack/salt/boss-tools/bossutils.sls

```diff
@@ -47,3 +47,11 @@ logging-critical:
         - group: www-data
         - makedirs: True
         - mode: 775
+
+# Allow unpriviledged scripts to create
+# the boss.config file
+boss-config:
```

Comment:
Is this only required for one ec2 instance or all of them? If the former, we could target a separate salt script for that one instance type.

Reply:
Just the one instance. Made this a separate SLS so only machines that need to can enable it.